plugin-api/src/main/java/org/hyperledger/besu/plugin/services/query/BftQueryService.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Collection;
 
 /** Allows for the IBFT 2.0 specific aspects of the block chain to be queried. */
-public interface IbftQueryService extends PoaQueryService {
```

Comment:
We should leave the IbftQueryService as is and mark it deprecated and add the BftQueryService. Perhaps the PoAMetricsService can now finally be removed now too.

Reply:
Flagged as deprecated, but would prefer to remove the PoAMetricsService in a separate PR